src/components/Form.js

```diff
@@ -32,8 +32,16 @@ const Form = () => {
 
     <input type='text' onInput={ titleInput } className="inputBook" placeholder="Title" value={title}></input>
     <input type='text' onInput={ authorInput } className="inputAuthor" placeholder="Author" value={author}></input>
-    <input type='text' onInput={ categoryInput } className="inputAuthor" placeholder="category" value={category}></input>
-
+    <select className="inputAuthor" onInput={ categoryInput } required>
+            <option value="" hidden>Category</option>
+            <option value="Action">Action</option>
+            <option value="Suspense">Suspense</option>
+            <option value="Romance">Romance</option>
+            <option value="Comedy">Comedy</option>
+            <option value="Thriller">Thriller</option>
+            <option value="Fiction">Fiction</option>
+            <option value="Real Life">Real Life</option>
+          </select>
   <button className="addBook" onClick={addBookButton}>ADD BOOK</button>
         </form>
```

Comment:
- Hey, Please let's also add the missing header text in the form to match the design as much ass we can below


*original design*
![Screenshot from 2022-11-19 05-19-05](https://user-images.githubusercontent.com/59397159/202833954-023d4061-e681-4f81-bb28-d69a2c0f6488.png)

*your design*
![Screenshot from 2022-11-19 05-18-36](https://user-images.githubusercontent.com/59397159/202833955-ac193868-fa46-4b5d-92ed-9c383d79922a.png)